neusomatic/python/generate_dataset.py

```diff
@@ -464,44 +464,47 @@ def prepare_info_matrices_tabix(
         for iii in range(len(tag_normal_matrices_)):
             tag_normal_matrices_[iii][0, :] = np.max(tag_normal_matrices_[iii])
 
-    (
-        tumor_matrix_,
-        bq_tumor_matrix_,
-        mq_tumor_matrix_,
-        st_tumor_matrix_,
-        lsc_tumor_matrix_,
-        rsc_tumor_matrix_,
-        tag_tumor_matrices_,
-        normal_matrix_,
-        bq_normal_matrix_,
-        mq_normal_matrix_,
-        st_normal_matrix_,
-        lsc_normal_matrix_,
-        rsc_normal_matrix_,
-        tag_normal_matrices_,
-        ref_array,
-        col_pos_map,
-    ) = align_tumor_normal_matrices(
-        record,
-        tumor_matrix_,
-        bq_tumor_matrix_,
-        mq_tumor_matrix_,
-        st_tumor_matrix_,
-        lsc_tumor_matrix_,
-        rsc_tumor_matrix_,
-        tag_tumor_matrices_,
-        tumor_ref_array,
-        tumor_col_pos_map,
-        normal_matrix_,
-        bq_normal_matrix_,
-        mq_normal_matrix_,
-        st_normal_matrix_,
-        lsc_normal_matrix_,
-        rsc_normal_matrix_,
-        tag_normal_matrices_,
-        normal_ref_array,
-        normal_col_pos_map,
-    )
+    if not tumor_col_pos_map == normal_col_pos_map:
```

Comment:
@pavanstrand we can't say `tumor_col_pos_map != normal_col_pos_map`? I am not sure if the "!=" operator works as the complement of `==` operator.

Reply:
I removing these changes